src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+    static String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
+        String [] b = {};
```

Comment:
what is this variable, what's it hold? I can't tell based on the name

Reply:
Fixed.